### DISCUSSION TITLE
Comments plugin usage

### DISCUSSION THREAD

**Discussion by muksidin** [Budibase/budibase#9070] [Help]
Hi.. any tutorial how to use comments plugin?
i 'm still confused.
Thanks.

**Comment by andz-bb** [comment]
hey @muksidin - there's a [readme for the plugin here](https://github.com/aptkingston/budibase-comment-box#budibase-comment-box) with some usage tips. Assuming you were able to install the plugin OK, it should just be a case of having a table with a text column for the comments to be stored, and then adding an empty row to that table and using that row's id in the component configuration.

<img width="1690" alt="image" src="https://user-images.githubusercontent.com/110921612/208086928-48f038e5-770a-4598-b4af-684fb4eb4520.png">

<img width="1379" alt="image" src="https://user-images.githubusercontent.com/110921612/208086797-022373ea-aa56-4060-b7f9-5606ae0ee62c.png">

Hope this helps!

**Reply by muksidin** [reply]
Thanks @andz-bb for your quick respond.
But i'm still not understand how to use that plugin. Can you explain step by step how to use it?
I want to use it for comment in data/record.
